html-block-and-inline/html-block-and-inline/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html><!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Block and Inline</title>
+  </head>
+  <body>
+    <h1>Blue Americana Snuggie&reg; Blanket</h1>
+    <h2>by SnuggieStore</h2>
+    <div>
+      <p>Price: <s>$19.99</s> $14.99</p>
+    </div>
+    <div>
+      <p>
+        Part of our new Americana collection, this <i>beautiful</i> blue is
+        already a customer favorite!
+      </p>
+    </div>
+    <div>
+      <p>Enjoy complete <b>Freedom of Movement!</b></p>
```

Comment:
same here, we should be using `<strong>` instead of `<b>`

Reply:
![Capture](https://github.com/aehrenkranz/m1-0723-code-solutions/assets/130402253/1d2e9d56-0e07-4323-92c5-4f56b5e688a6)

Updated file